src/v2/Components/v2/ArtworkFilter/ArtworkFilters/WaysToBuyFilter.tsx

```diff
@@ -54,8 +58,10 @@ export const WaysToBuyFilter: FC = () => {
     },
   ]
 
+  const isExpanded = expanded
```

Comment:
Thought: if renaming/reassigning this var numerous times across files throughout the PR, maybe refactor the prop name itself to `isExpanded`? Creating intermediate vars like this creates a sort of 1:2 complexity scenereo, where the contract (interface) diverges from the implementation (local variable) leading to a tiny amount of friction/indirection that adds up at scale.  

But that said, if `<Toggle>` expects `expanded` and the filter inteface is refactored to be `isExpanded`, thereby eliminating the local var, you've still got a 1:2 scenereo with `<Toggle expanded={isExpanded}>`. Best if everything is consistent everywhere, even if prefixing a boolean with `is` is slightly more idomatic.

Reply:
Thanks for checking out my PR. I was trying to see if there was a way to "inject" the expand/collapse behavior into each filter, and `isExpanded` was my first step towards standardizing how each filter determines that behavior. (Some filters use the `expanded` prop directly, others combine it with logic to determine if there are filter options to display.)

I eventually want something like an `Expandable` interface that provides an `expanded` prop and an `isExpanded()` method that filters can override if there is additional criteria to consider when deciding to expand or collapse a toggle. Does that sound feasible?

Reply:
> isExpanded was my first step towards standardizing how each filter determines that behavior

I see, this is a sort of incremental step where some future commit would roll this stuff up. Sorry, jumped the gun ini a draft pr! 🤦(Though I also never tire of raising 1:1 vs 1:2 complexity points, heh) 

Re `Expandable`,  @dzucconi just implemented it over in Palette: https://github.com/artsy/palette/pull/896

Reply:
I don't think there needs to be an `isExpanded` method (we don't have methods anyway); just pass that criteria into the `expanded` prop. Though right now I think we have do a minor update to it so that it responds to changes in the prop, not just the initial mount.

Reply:
The situation I was thinking of is [this one](https://github.com/artsy/force/blob/06a14410a5015b10031e07c9a310093268856953/src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ColorFilter.tsx#L105) where the `ColorFilter` component takes in an `expanded` prop but can also preempt it if there are no colors to display.

I'm OK removing `isExpanded` though because I don't see a way forward with what I want to do with it.